compile/src/main/scala/sbt/LoggerReporter.scala

```diff
@@ -93,7 +93,8 @@ class LoggerReporter(maximumErrors: Int, log: Logger, sourcePositionMapper: Posi
       log(msg)
     else {
       val sourcePrefix = m2o(pos.sourcePath).getOrElse("")
-      val lineNumberString = m2o(pos.line).map(":" + _ + ":").getOrElse(":") + " "
+      val columnNumber = m2o(pos.pointer).map(_.toInt + 1).getOrElse(1)
```

Comment:
This took me also by surprise, but in this way it is consistent with other tools like [the_silver_searcher](https://github.com/ggreer/the_silver_searcher) which count columns from 1, for example: 

```
$ ag --nogroup --line-number --column -- package .
main/src/main/scala/sbt/internal/BuildUtil.scala:1:1:package sbt
main/src/main/scala/sbt/internal/BuildUtil.scala:2:1:package internal
```

Also vim seems to count them from 1, in emacs land it seems little [inconsistent](https://github.com/bbatsov/rubocop/issues/276), but `next-error` command counts them also from 1.